appcues/src/main/java/com/appcues/trait/BackdropDecoratingTrait.kt

```diff
@@ -6,5 +6,5 @@ import androidx.compose.runtime.Composable
 interface BackdropDecoratingTrait : ExperienceTrait {
 
     @Composable
-    fun BoxScope.Backdrop()
+    fun BoxScope.BackdropDecorate(content: @Composable BoxScope.() -> Unit)
```

Comment:
no issue at all as its necessary for our project, but this might mark our official break to 2.0 (first public API change)